templates/repo/issue/view_content/comments.tmpl

```diff
@@ -1,802 +1,804 @@
 {{ template "base/alert" }}
 {{range .Issue.Comments}}
-	{{ $createdStr:= TimeSinceUnix .CreatedUnix $.Lang }}
+	{{if call $.IsEventShown .Type}}
```

Comment:
I'm not sure if `call` is necessary

Reply:
Tried this and it's not working without `call`